crates/typst-library/src/text/mod.rs

```diff
@@ -364,6 +364,17 @@ pub struct TextElem {
     #[resolve]
     pub hyphenate: Hyphenate,
 
+    /// The "cost" of hyphenation when laying out text.
+    /// A higher cost means the layout engine will hyphenate less often.
+    ///
+    /// When justifying text, a higher hyphenation cost can result in more variation
+    /// in justification spacing in order to avoid hyphenation.
+    ///
+    /// The default cost is an acceptable balance, but some may find that it hyphenates too eagerly.
+    /// A cost of `10.0` may work better for such contexts.
+    #[default(0.5)]
```

Comment:
Is it possible to deduplicate this value with the `const DEFAULT_HYPH_COST` from the other file?

Reply:
Yeah, I was confused about that. `shared_get` returns an Option despite this having a default value. I'm not very familiar with the whole system.

Reply:
shared_get returns `None` if the value is not uniform for the whole paragraph

Reply:
So what is the solution there? Is it correct to default to the default cost when shared_get returns None?